<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arreglos</class>
 <widget class="QDialog" name="Arreglos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushAgregar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Append</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Posicion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBuscar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>40</y>
     <width>101</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinInsertarPos"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinEliminar"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSpinBox" name="spinAgregar"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinInsertarNum"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBuscar"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushInsertar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushEliminar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultBuscar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Numero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushClear">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
